pkg/scheduler/plugins/joborder/job_order.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package joborder
```

Comment:
I think the plugin name is a bit misleading, maybe it should be `reflectjoborder` or something similar?